pypit/_version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = '0.6.dev0'
```

Comment:
OK -- you've sold me!

On 9 June 2016 at 22:27, J. Xavier Prochaska <notifications@github.com>
wrote:

> In pypit/_version.py
> <https://github.com/PYPIT/PYPIT/pull/84#discussion_r66525153>:
>
> > @@ -0,0 +1,2 @@
> > +__version__ = '0.6.dev0'
>
> To be honest, I copied the DESI approach. Not sure why
> they named it this way.
>
> According to the great Google, this is the preferred way:
>
>
> http://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/PYPIT/PYPIT/pull/84/files/c1982c97f902903e045bdd14d9e33c49f3ac9497#r66525153>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMALHtXAD6kkbucczjAhVQsPtJVn18pDks5qKIVTgaJpZM4IxqGg>
> .
>